src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -39,15 +42,19 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the property identified "
             + "by the index number used in the displayed property list. "
-            + "Existing values will be overwritten by the input values.\n"
+            + "Existing values will be overwritten by the input values, with the exception of adding tags. "
+            + "Added tags will be appended to the current tags. \n"
             + "Parameters: INDEX (must be a positive integer) "
             + "[" + PREFIX_NAME + "NAME] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
             + "[" + PREFIX_PRICE + "PRICE] "
             + "[" + PREFIX_SELLER + "NAME] "
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "[" + PREFIX_TAG + "TAG]..."
+            + "[" + PREFIX_ADD_TAG + "TAG]..."
+            + "[" + PREFIX_DELETE_TAG + "TAG]..."
```

Comment:
Would changing `"TAG"` to `"TAG_TO_ADD"` make it clearer?
This is the current help message:
`Parameters: INDEX (must be a positive integer) [n/NAME] [a/ADDRESS] [$/PRICE] [s/NAME] [p/PHONE] [e/EMAIL] [t/TAG]...[ta/TAG]...[td/TAG]...`